src/irmin-pack/unix/ext.ml

```diff
@@ -399,9 +407,9 @@ module Maker (Config : Conf.S) = struct
             Contents.CA.purge_lru t.contents;
             Node.CA.purge_lru t.node;
             Commit.CA.purge_lru t.commit;
-            let span2 = Mtime_clock.count c0 |> Mtime.Span.to_us in
+            let span2 = Mtime_clock.count c0 |> Mtime.Span.to_s in
             [%log.debug
-              "Gc swap and purge: %.0fus, %.0fus" span1 (span2 -. span1)];
+              "Gc swap and purge: %.0fs, %.0fs" span1 (span2 -. span1)];
```

Comment:
you might want to change the rounding here otherwise `%.0f` will always display 0 or 1.